script/Arcueid/Day 7/07_00_ARC07_1C.txt

```diff
@@ -252,7 +252,7 @@ I'm at a loss for words.
 [sha:9e05c1e0648b8d50c41c58296b29837e1a61e188]{
 -- Page 108, Offset 16838.
 -- あいつは。ヴローヴは死んだ。この手で殺した。
-That guy is――― Vlov is dead. I killed him with my own hands.
+That guy――Vlov――is dead. I killed him with my own two hands.
```

Comment:
When surrounded by '――' I feel as though there should be a space before the word. Maybe its my malfuction to try to read it fast if there is no space there though.

Reply:
These CJK dashes are stand-ins for em-dashes. You don't typically put spaces surrounding em-dashes.